### DISCUSSION TITLE
Storing Multi-Band Multi-Time Remote Sensing Data Using JPEG XL Animation

### DISCUSSION THREAD

**Discussion by JulioContrerasH** [libjxl/libjxl#3765] [Q&A]
I'm working in remote sensing, where it's common to handle images with more than 3 bands (usually in uint16 format), which require lossless compression. We have images from the same region of the planet stored in JPEGXL format almost all the images looks quite similar, and we want to store them efficiently. Knowing that JPEG XL supports animations, would it be possible to store our data cube (TIME = 20 frames, CHANNELS = 10 channels, HEIGHT = 256, WIDTH = 256) in uint16, losslessly, as a JPEG XL animation?

**Comment by zougloub** [comment]
Just commenting to be in the loop... this is even broader than discussions on efficient hyperspectral compression, touched in https://github.com/libjxl/libjxl/issues/1245.
